content/working_with/console/widgets/deploymentsView.md

```diff
@@ -167,7 +171,8 @@ customized using the
 Drilling down can happen multiple times if there are more levels in the
 deployments hierarchy.
 
-To go back to the parent page, use the breadcrumbs in top part of the page.
+To go back to the parent page, use "Parent" button in the details pane header
```

Comment:
Would `details pane header` be clear enough for the user? 🤔 
As the word `details` is used here, it may be indicating a need for clicking` Details` button in order to see the `Parent` button 😁 

WDYT about renaming it to be something like 
```suggestion
To go back to the parent page, use "Parent" button which is placed near the deployment name
```

? 🤔

Reply:
I used details pane term as there's a section called the same. Added a link to that section to make it more clear. WDYT about that change?

Reply:
Oh, that explains a lot 😁 
Now it looks great :)

Thanks for the explaination and for the adjustment 🎸